worker/dbaccessor/manifold.go

```diff
@@ -166,3 +170,15 @@ func dbAccessorOutput(in worker.Worker, out interface{}) error {
 	}
 	return nil
 }
+
+// IAASNodeManager returns a NodeManager that is configured to use
+// the cloud-local TLS terminated address for Dqlite.
+func IAASNodeManager(cfg agent.Config, logger Logger, slowQueryLogger coredatabase.SlowQueryLogger) NodeManager {
```

Comment:
Then you wouldn't need these.

Reply:
I like these, it's very explicit that although temporary they're calling it out that there is a different setup between IAAS and CAAS. This is very much, dependency driven.